config/locales/home/habitat_cover/en.yml

```diff
@@ -10,28 +10,19 @@ en:
         coralreefs:
           title: Warm-water coral reefs
           text: ! '<strong>%{change_percentage}</strong> loss in warm-water coral cover<br>between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
-          citation_url: '<a href="%{citation_url}">%{citation_url}</a>'
         saltmarshes:
           title: Saltmarshes
           text: ! '<strong>%{change_percentage}</strong> loss in saltmarsh cover<br> between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
-          citation_url: '<a href="%{citation_url}">%{citation_url}</a>'
         mangroves:
           title: Mangroves
           text: ! '<strong>%{change_percentage}</strong> loss in mangrove cover<br> between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
-          citation_url: '<a href="%{citation_url}">%{citation_url}</a>'
         seagrasses:
           title: Seagrasses
           text: ! '<strong>%{change_percentage}</strong> loss in seagrass cover<br> between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
-          citation_url: '<a href="%{citation_url}">%{citation_url}</a>'
         coldcorals:
           title: Cold-water corals
           text: ! '<strong>%{change_percentage}</strong> loss in cold-water coral cover<br> between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
-          citation_url: '<a href="%{citation_url}">%{citation_url}</a>'
       data_deficient: Unfortunately, there is currently no data for this particular habitat.
+      no_change_percentage: '-'
```

Comment:
Given the previous comments, this shouldn't be needed anymore. Sorry, perhaps original it was meant to be a hyphen.